.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Why this file?

Reply:
was added automatically by latest vscode. Can remove but I suspect it will be added by someone else